app/src/ui/app.tsx

```diff
@@ -583,6 +585,23 @@ export class App extends React.Component<IAppProps, IAppState> {
     this.props.dispatcher.setAppMenuState(menu => menu.withReset())
   }
 
+  private onTitlebarDoubleClickDarwin() {
+    const actionOnDoubleClick: AppleActionOnDoubleClickPref = remote.systemPreferences.getUserDefault('AppleActionOnDoubleClick', 'string')
```

Comment:
How would y'all feel about enabling `noImplicitAny` in the TypeScript config in another PR? We have it enabled at work, and although it is more aggressive, it saves us from having to be human linters to catch cases like this (which was originally an implicit `any`)